hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/VolumeManager.java

```diff
@@ -141,4 +143,43 @@ boolean checkVolumeAccess(String volume, OzoneAclInfo userAcl)
    */
   List<OmVolumeArgs> listVolumes(String userName, String prefix,
       String startKey, int maxKeys) throws IOException;
+
+  /**
+   * Add acl for Ozone object. Return true if acl is added successfully else
+   * false.
+   * @param obj Ozone object for which acl should be added.
+   * @param acl ozone acl top be added.
+   *
+   * @throws IOException if there is error.
+   * */
+  boolean addAcl(OzoneObj obj, OzoneAcl acl) throws IOException;
```

Comment:
We don't need to pass OzoneObj here as a volume name is good enough for VolumeVanagerImpl to do the ACL operations. Some of the ACL type checking has already been done at OM.

Reply:
Since OzoneObj is wrapper remaining information might be used later to differentiate b/w stores. Also it keeps API consistent for all entities.